src/grid/cubic.py

```diff
@@ -0,0 +1,516 @@
+import numpy as np
+from scipy.interpolate import CubicSpline
+
+from grid.basegrid import Grid, OneDGrid
+
+
+class CubicGrid(Grid):
+    r"""
+
+    Attributes
+    ----------
+
+    Methods
+    -------
```

Comment:
We have not been listing Attributes/Methods in the docstrings, so don't worry about it.

Reply:
Resolved in d958179